automation/conformance.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -exuo pipefail
```

Comment:
I'm not a big bash expert, but it is written [here](https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/) that -E is required when you inherit a trap, so that the -e won't cause the script to exit before the trap is fired.

Reply:
just if running in a subshell, else its safe

Reply:
It also applies to called shell functions. We may not need it in the current state of our scripts but it is a safe option to add it.

Done

Reply:
fyi, on other PRs it was concluded that it does work on shell functions (we even binded to bash here), when i raised it (with the same url even).
up to you, i just think we should line up on a method.

Reply:
Ok, then let's keep it in case we ever introduce a subshell to the script. Was that tested by you too?

Reply:
not in that PR mentioned scope, it was tested ad hoc by the author.

Reply:
subshell of shell function?

Reply:
he tested both:
* sub shell - needs -E
* shell function (he used bash from what i remember) - doesnt need -E

Reply:
Who is "he", do you have a reference?

Reply:
if i find the threads i will paste

Reply:
shell functions
kubevirt/kubevirtci#428 (comment)

subshell
kubevirt/kubevirtci#428 (comment)

Reply:
You linked the PR twice, could you give me link to the specific comment?

Reply:
copy paste removed the link
shell functions (whole discussion, bottom line in the end)
https://github.com/kubevirt/kubevirtci/pull/428#discussion_r476229164

subshell (bottom line)
https://github.com/kubevirt/kubevirtci/pull/428#discussion_r476558135

Reply:
In the second link, @ormergi commented that -E **is** needed. Am I reading it wrong?

Reply:
in subshell only, see first that say its not needed in function (see bottom of first thread please)
exactly as i said above.

Reply:
So I want to keep -E in case anybody ever introduces a subshell into one of these scripts.

Sounds good to you?

Reply:
yes thx, with the only opinion that we should stick to it in my opinion on every PR we are creating, 
its easier, safer and saves time
(i understand some might think otherwise)

else i dont see why to put stuff for future, unless its per Author decision, which might waste time on future discussion.

up to you, thx

Reply:
I think of this as a valid reset to make Bash work in an expected manner, no matter if all the code in the scripts needs it or not.

If you are ok with this shape, consider granting lgtm.